fix(event): update text and scholarship dates (#493)
pyOpenSci/pyopensci.github.io: _posts/events/2024-08-09-pyopensci-fall-festival.md

</div>

**Important:** The agenda items in EventBrite are listed in UTC-6:00- This is an EventBrite bug. If you need clarification on start times, please email us at [media@pyopensci.org](mailto:media@pyopensci.org).
{: .notice .notice--primary }

### Pricing tiers

We've created several pricing tiers to make the Fall Festival accessible to as many people as possible.

#### Day one - October 28, 2024: Keynote talks

Keynote talks are free for anyone to attend. Register now before we fill up!

#### Days 2-4 - 29 October - November 1, 2024: Technical Workshops

You can register for a single-day workshop or the full event. Workshop registration includes office hours each day for more one-on-one help in the afternoon.

| Registration Type  | Cost |
|----------|----------|
| Day One Keynote Talks - October 28 2024 | Free | |
| Full 4-day Workshop: Student | $350 | |
| Full 4-day Workshop: Standard | $475 | |
| Full 4-day Workshop: Corporate | $625 | |
|  |  | |
| Single Day Workshop: Student | $100 | |
| Single Day Workshop: Standard | $150 | |
| Single Day Workshop: Corporate | $200 | |

### Scholarships

> If you are financially unable to attend pyOpenSci's Fall Festival, you can apply for a scholarship using [this form](https://www.surveymonkey.com/r/pyOSFF2024). We will accept applications until Monday, September 30. Application review will begin on  Friday, October 11 2024.

Author: ```suggestion
If you are financially unable to attend pyOpenSci's Fall Festival, you can apply for a scholarship using [this form](https://www.surveymonkey.com/r/pyOSFF2024). We will accept applications until Monday, October 7, 2024. Application review will begin on Friday, October 11 2024.
```

* **What:** An online community training and networking event
* **Where:** Online using [Gather](https://www.gather.town/) (learn more about Gather's system requirements [here](https://support.gather.town/hc/en-us/articles/15909789090452-System-Requirements))
* **When:** Monday, October 28--Friday, November 01, 2024
* **Who:** If you are a scientist, a researcher, a student, or anyone interested in writing better, cleaner code that can be installed into different environments and shared, then this event is for you!
* **Cost - Day 1 Keynote talks:** free.
* **Cost- workshops:** $350--$625, with scholarships available

> * [**pyOpenSci demo package**](https://github.com/pyOpenSci/pyosPackage)

Author: ```suggestion

```

## pyOpenSci's Fall Festival registration, cost, and scholarship information

<div style="text-align: center;" markdown="1">

[--> Click Here to Register now for the pyOpenSci Open Science Fall Festival <--](https://www.eventbrite.com/e/pyopensci-fall-festival-2024-registration-1000762657047){: .btn .btn--success .btn--large }
</div>

**Important:** The agenda items in EventBrite are listed in UTC-6:00- This is an EventBrite bug. If you need clarification on start times, please email us at [media@pyopensci.org](mailto:media@pyopensci.org).
{: .notice .notice--primary }

### Pricing tiers

We've created several pricing tiers to make the Fall Festival accessible to as many people as possible.

#### Day one - October 28, 2024: Keynote talks

Keynote talks are free for anyone to attend. Register now before we fill up!

#### Days 2-4 - 29 October - November 1, 2024: Technical Workshops

You can register for a single-day workshop or the full event. Workshop registration includes office hours each day for more one-on-one help in the afternoon.

| Registration Type  | Cost |
|----------|----------|
| Day One Keynote Talks - October 28 2024 | Free | |

> | Full Workshop: Student | $350 | |
> | Full Workshop: Standard | $475 | |
> | Full Workshop: Corporate | $625 | |

Author: ```suggestion
| Full 4-day Workshop: Student | $350 | |
| Full 4-day Workshop: Standard | $475 | |
| Full 4-day Workshop: Corporate | $625 | |
```

Quarto is an open source tool based on Pandoc that allows you to create and publish reproducible, production-quality articles, presentations, dashboards, websites, blogs, and books in HTML, PDF, MS Word, ePub, and more, right from your Jupyter notebooks.

With Great Tables you can make wonderful-looking tables in Python. Great Tables is an open source Python package that lets you mix and match things like a header and footer, attach a stub (which contains row labels), arrange spanner labels over top of the column labels, and much more. Not only that, but you can format the cell values in a variety of awesome ways.

## Speakers and instructors

> _We'll continue to update this section with more information as we confirm speakers and instructors for the event!_

Author: ```suggestion
*We'll continue to update this section with more information as we confirm speakers and instructors for the event!*
```
